### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,43 @@ All notable changes to this project will be documented in this file.
...
+
+**Merged pull requests:**
+
+- Bump nokogiri from 1.14.2 to 1.14.3 [\#150](https://github.com/betadots/hdm/pull/150) ([dependabot[bot]](https://github.com/apps/dependabot))
+- Update to Ruby 3.2.2 [\#149](https://github.com/betadots/hdm/pull/149) ([rwaffen](https://github.com/rwaffen))
+- Show unused environments and allow matching nodes to other environments [\#146](https://github.com/betadots/hdm/pull/146) ([oneiros](https://github.com/oneiros))
+- Bump rack from 2.2.6.3 to 2.2.6.4 [\#145](https://github.com/betadots/hdm/pull/145) ([dependabot[bot]](https://github.com/apps/dependabot))
+- update rails gems [\#144](https://github.com/betadots/hdm/pull/144) ([rwaffen](https://github.com/rwaffen))
+- Bump rack from 2.2.6.2 to 2.2.6.3 [\#143](https://github.com/betadots/hdm/pull/143) ([dependabot[bot]](https://github.com/apps/dependabot))
+- Initial support for SAML SSO \#95 [\#142](https://github.com/betadots/hdm/pull/142) ([oneiros](https://github.com/oneiros))
```
Comment (bastelfreak) [inline CHANGELOG.md:33]: should we label that as enhancement?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- Deep merge hiera defaults \#108 [\#141](https://github.com/betadots/hdm/pull/141) ([oneiros](https://github.com/oneiros))
```
Comment (bastelfreak) [inline CHANGELOG.md:34]: should this be an enhancement or bug(fix)?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,12 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- Bump docker/build-push-action from 3 to 4 [\#139](https://github.com/betadots/hdm/pull/139) ([dependabot[bot]](https://github.com/apps/dependabot))
+
+## [v1.0.1](https://github.com/betadots/hdm/tree/v1.0.1) (2023-01-31)
+
+[Full Changelog](https://github.com/betadots/hdm/compare/v1.0.0...v1.0.1)
 
 **Merged pull requests:**
 
+- Release 1.0.1 [\#138](https://github.com/betadots/hdm/pull/138) ([rwaffen](https://github.com/rwaffen))
```
Comment (bastelfreak) [inline CHANGELOG.md:43]: I added the skip-changelog label

**Review by tuxmea — DISMISSED** [review]
OK for me. Maybe check Tims comments.

File: CHANGELOG.md (hunk 1 above)
Comment (rwaffen) [reply CHANGELOG.md:34]: all 4 or a specific one?

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**